CMakeLists.txt

```diff
@@ -177,7 +177,7 @@ set_target_properties(obs-browser-page PROPERTIES FOLDER "plugins/obs-browser")
 
 if (APPLE)
 	set_target_properties(obs-browser-page PROPERTIES
-		COMPILE_FLAGS "-mmacosx-version-min=10.8")
+		COMPILE_FLAGS "-mmacosx-version-min=10.13")
```

Comment:
Could you remove this change so that we can retain the discussions in #233?

Reply:
Removed and force-pushed an update.